cmd/mistry/main.go

```diff
@@ -127,6 +130,11 @@ EXAMPLES:
 					Usage:       "rebuild the docker image",
 					Destination: &rebuild,
 				},
+				cli.StringFlag{
+					Name:        "timeout",
+					Usage:       "time to wait for the build to finish, accepts values as defined at https://golang.org/pkg/time/#ParseDuration",
```

Comment:
We could save the user from having to open a URL in order to use mistry. For this, we could introduce a reasonable default (60mins.) so that it's included in the help output.

> Maximum time to wait for a build.